input/resources/StructureDefinition-profile-patient.json

```diff
@@ -44,6 +44,216 @@
+        "path": "Patient.identifier.type.coding.display",
+        "fixedString": "Medicaid ID"
+      },
+      {
+        "id": "Patient.identifier:MedicaidID.type.text",
+        "path": "Patient.identifier.type.text",
+        "definition": "A human language representation of the concept as seen/selected/uttered by the user who entered the data and/or which represents the intended meaning of the user.",
+        "fixedString": "Patient Medicaid number"
+      },
+      {
+        "id": "Patient.identifier:MedicaidID.system",
+        "path": "Patient.identifier.system",
+        "fixedUri": "https://signalbhn.org/fhir/sid/us-medicaidid"
+      },
+      {
+        "id": "Patient.identifier:MedicareID",
+        "path": "Patient.identifier",
+        "sliceName": "MedicareID",
+        "short": "Medicare identifier for this patient",
+        "definition": "Medicare identifier for this patient.",
+        "max": "1"
+      },
+      {
+        "id": "Patient.identifier:MedicareID.type.coding.system",
+        "path": "Patient.identifier.type.coding.system",
+        "fixedUri": "https://www.hl7.org/fhir/valueset-identifier-type.html"
+      },
+      {
+        "id": "Patient.identifier:MedicareID.type.coding.code",
+        "path": "Patient.identifier.type.coding.code",
+        "fixedCode": "MR"
+      },
+      {
+        "id": "Patient.identifier:MedicareID.type.coding.display",
+        "path": "Patient.identifier.type.coding.display",
+        "fixedString": "Medicare ID"
+      },
+      {
+        "id": "Patient.identifier:MedicareID.type.text",
+        "path": "Patient.identifier.type.text",
+        "fixedString": "Patient Unique Medicare Number"
+      },
+      {
+        "id": "Patient.identifier:MedicareID.system",
+        "path": "Patient.identifier.system",
+        "fixedUri": "http://hl7.org/fhir/sid/us-mbi"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId",
+        "path": "Patient.identifier",
+        "sliceName": "ProviderClientId",
+        "short": "ProviderClient identifier for this patient",
+        "definition": "ProviderClient identifier for this patient.",
+        "max": "1"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId.type.coding.system",
+        "path": "Patient.identifier.type.coding.system",
+        "fixedUri": "https://www.hl7.org/fhir/valueset-identifier-type.html"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId.type.coding.code",
+        "path": "Patient.identifier.type.coding.code",
+        "fixedCode": "MR"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId.type.coding.display",
+        "path": "Patient.identifier.type.coding.display",
+        "fixedString": "Medical Record Number"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId.system",
+        "path": "Patient.identifier.system",
+        "fixedUri": "PROVIDER SYSTEM URL"

```

Comment:
Do not hard code this value.  The developers will have to provide this when they input a ProviderClientID
Add a note in the "Definition" field:  "This needs to be the URL that is specific to the Provider's domain; e.g. http://example.org"

Reply:
Removed System -->Fixed value and added definition as mentioned
![image](https://github.com/enjoysparkling/signal-mso-fhir-profiles/assets/136109806/5d0ae0c5-4cf3-4a32-bb7d-37f20e84d0d7)

Reply:
I do not see the Definition on Patient.identifier:ProviderClientId.system.  Would you double-check that it saved properly?